habitat_baselines/common/tensorboard_utils.py

```diff
@@ -10,6 +10,20 @@
 import torch
 from torch.utils.tensorboard import SummaryWriter
 
+try:
+    import wandb
+except ImportError:
+    pass
+
+
+def get_writer(config, **kwargs):
+    if config.WRITER_TYPE == "tb":
+        return TensorboardWriter(config.TENSORBOARD_DIR, **kwargs)
+    elif config.WRITER_TYPE == "wb":
+        return WeightsAndBiasesWriter(config)
```

Comment:
Watch out, I added this code on a previous commit and it is not very well tested.

Reply:
Ok, removed it since it is not necessary for this PR